webapp/channels/src/components/widgets/menu/menu_modals/submenu_modal/submenu_modal.tsx

```diff
@@ -17,63 +17,58 @@ type Props = {
     onExited: () => void;
 }
 
-type State = {
-    show: boolean;
-}
-
-export default class SubMenuModal extends React.PureComponent<Props, State> {
-    public constructor(props: Props) {
-        super(props);
-        this.state = {
-            show: true,
-        };
-    }
+const SubMenuModal = ({
+    elements,
+    onExited,
+}: Props) => {
+    const [show, setShow] = useState(true);
 
-    public onHide = () => { //public because it is used on tests
-        this.setState({show: false});
+    const onHide = () => {
+        setShow(false);
```

Comment:
To avoid passing the a new function on every render, let's use a `useCallback` here.

Reply:
Done